Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -495,12 +496,20 @@ mode would look as follows:
 Socket LoadBalancer Bypass in Pod Namespace
 *******************************************
 
+The socket-level loadbalancer acts transparent to Cilium's lower layer datapath
+in that upon ``connect`` (TCP, connected UDP), ``sendmsg`` (UDP), or ``recvmsg``
+(UDP) system calls, the destination IP is checked for an existing service IP and
+one of the service backends is selected as a target. This means that although
+the application assumes it is connected to the service address, the
+corresponding kernel socket is actually connected to the backend address and
+therefore no additional lower layer NAT is required.
```

Comment:
This is a very succinct and understandable explanation of socket LB, thanks for adding this paragraph!